src/main/java/io/apisense/embed/influx/configuration/InfluxConfigurationWriter.java

```diff
@@ -1,13 +1,7 @@
 package io.apisense.embed.influx.configuration;
 
 import com.moandjiezana.toml.TomlWriter;
-import io.apisense.embed.influx.configuration.server.ConfigurationProperty;
-import io.apisense.embed.influx.configuration.server.ConfigurationSection;
-import io.apisense.embed.influx.configuration.server.DataConfigurationSection;
-import io.apisense.embed.influx.configuration.server.HeadConfigurationSection;
-import io.apisense.embed.influx.configuration.server.HttpConfigurationSection;
-import io.apisense.embed.influx.configuration.server.MetaConfigurationSection;
-import io.apisense.embed.influx.configuration.server.UdpConfigurationSection;
+import io.apisense.embed.influx.configuration.server.*;
```

Comment:
Do not use the star import as it often import much more than necessary.

Reply:
It was done automatically, I will revert such changes.